java/lib.jshell.agent/agentsrc/org/netbeans/lib/jshell/agent/LoaderDelegator.java

```diff
@@ -42,4 +42,7 @@ public void addToClasspath(String string) throws ExecutionControl.EngineTerminat
     public Class<?> findClass(String string) throws ClassNotFoundException {
         throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
     }
+    public void classesRedefined(ExecutionControl.ClassBytecodes[] cbcs) {
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
Please change the body of the method to something meaninful.

Reply:
Even other methods  defined in the class are throwing UnsupportedOperationException. I  believe  the change should be fine.